vowpalwabbit/parse_args.cc

```diff
@@ -288,11 +281,10 @@ void parse_dictionary_argument(vw& all, std::string str)
 
   if (!all.quiet)
     all.trace_message << "dictionary " << s << " contains " << map->size() << " item" << (map->size() == 1 ? "" : "s")
-                      << endl;
+                      << std::endl;
```

Comment:
Can we get rid of the many std::endl with a 'using' declaration?  It seems worthwhile in this file.

Reply:
I can do that